src/Psl/Collection/MutableVector.php

```diff
@@ -69,6 +70,20 @@ public static function fromArray(array $elements): MutableVector
         return new self($elements);
     }
 
+    /**
+     * Create a vector from the given $items iterable.
+     *
+     * @template Ts
+     *
+     * @param iterable<Ts> $items
+     *
+     * @return MutableVector<Ts>
+     */
+    public static function fromItems(iterable $items): MutableVector
```

Comment:
could you please add the same method to other collection types? ( `Vector`, `Set`, `MutableSet`, `Map`, and `MutableMap` ).

Reply:
Maybe `fromIterable()` would be more meaningful?

Reply:
Done! Also added an alias for `containsKey` for parity with hhvm collections